sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdkProvider.java

```diff
@@ -119,6 +119,15 @@ public void shutdown() {
     sharedState.stop();
   }
 
+  /**
+   * Requests the active span processor to process all span events that have not yet been processed.
+   *
+   * @see SpanProcessor#forceFlush()
+   */
+  public void forceFlush() {
+    sharedState.getActiveSpanProcessor().forceFlush();
+  }
+
```

Comment:
Do we need this on the SDK? Probably yes, but would like to understand the motivation you had.

Reply:
See https://github.com/open-telemetry/opentelemetry-java/pull/1068#issuecomment-607932185

Reply:
This is also in the Specification ;)

Reply:
Ah, makes sense!

Reply:
@carlosalberto Actually it's not, only shutdown is: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md#shutdown-1. Also forceFlush is not on the exporter. I'll try creating a PR.

Reply:
Reading more carefully, it seems that the spec does not even specify shutdown on the TracerProvider either, only the getTracer method in the API.

Reply:
It's only in the spec for a SpanProcessor, not for the SDK Tracer implementation, unless I missed that somewhere.

Reply:
Oh, right. I misread that one. In any case, I think it makes sense to have in the `Tracer` SDK section. I don't imagine keeping the processor object around for this specific task.

Reply:
@bogdandrutu any more clarification needed on this?